app/api/call_for_papers.py

```diff
@@ -56,17 +56,6 @@ def before_create_object(self, data, view_kwargs):
             event = self.session.query(Event).filter_by(id=view_kwargs['event_id']).one()
             data['event_id'] = event.id
 
-    def after_get(self, result):
```

Comment:
Yeah, that should be added when the migrations for making the date timezone naive again are made.